File: example3/src/FibonacciSeries.cpp

#include <iostream>
using namespace std;

void printFibonacciSeries(int n) {
    int n1 = 0, n2 = 1, n3;
    cout << n1 << " " << n2 << " ";
    
    for(int i = 2; i < n; ++i) {
        n3 = n1 + n2;
        cout << n3 << " ";
        n1 = n2;
        n2 = n3;
    }
    cout << endl;
}

int main() {
    int count = 10;
    cout << "Fibonacci Series of " << count << " numbers: ";
    printFibonacciSeries(count);
    return 0;
}

File: example3/src/FibonacciSeries.java

public class FibonacciSeries {
    public static void printFibonacciSeries(int count) {
        int n1 = 0, n2 = 1, n3;
        System.out.print(n1 + " " + n2); //printing 0 and 1    
        
        for (int i = 2; i < count; ++i) {
            n3 = n1 + n2;
            System.out.print(" " + n3);
            n1 = n2;
            n2 = n3;
        }
    }
    
    public static void main(String[] args) {
        int count = 10;
        System.out.print("Fibonacci Series of " + count + " numbers: ");
        printFibonacciSeries(count);
    }
}